src/widgets/modalChildren/NewConfirmModal.js

```diff
@@ -76,8 +76,6 @@ export const NewConfirmModal = props => {
   );
 };
 
```

Comment:
(I know this isn't part of the refactor! Just mentioning it) 
I'm not sold on the name. Is this actually a `ChildConfirmModal`? Is the intention to replace the `ConfirmModal` with the new one, or have I missed something?

Reply:
Sorry, yes the intention is to replace `ConfirmModal` with this one, am making an issue right now!

I forgot to mention I hate the `modalChildren` name if you have any other ideas..